### PR TITLE
feat(node): update npm-shrinkwrap file

### LINKED ISSUE #886: feature: support updating npm-shrinkwrap.json
**Is your feature request related to a problem? Please describe.**

Release please PR doesn't update `npm-shrinkwrap.json`. It does update `package-lock.json`

 **Describe the solution you'd like**

Release please should detect the `npm-shrinkwrap.json` file and update its version

 **Describe alternatives you've considered**

Modifying the release PR manually.

### PR DESCRIPTION
**Pull request by erezrokah**
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/release-please/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #886
Fixes https://github.com/netlify/cli/issues/2082

> Opening in draft until I add some tests

### CODE REVIEW

File: src/updaters/package-lock-json.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  version: string;
+  lockfileVersion: 2;
+  packages: Record<string, {version: string}>;
+};
+
+export class PackageLockJson extends PackageJson {
+  updateVersion(parsed: {version: string; lockfileVersion: number}) {
+    super.updateVersion(parsed);
+    if (parsed.lockfileVersion === 2) {
+      const lockFileV2 = parsed as LockFileV2;
```
Comment (erezrokah) [inline src/updaters/package-lock-json.ts:27]: See https://github.com/netlify/cli/pull/2263/files#diff-44a7c69871c5958e657226d5552c9451606a778233fb824f68530d0cfd3f8994R9 for version 2 of npm lock files

File: src/github.ts [hunk 2]
```diff
@@ -302,7 +302,7 @@ export class GitHub {
     let page = 1;
     let found = false;
     const baseBranch = await this.getDefaultBranch();
-    const commits = [];
+    const commits: [string | null, string][] = [];
```
Comment (erezrokah) [inline src/github.ts:305]: Without this part the compilation failed for me

**Comment by erezrokah** [comment]
Not sure what to do with the tests, as they fail for me on the default branch too.

**Review by bcoe — APPROVED** [review]

**Comment by bcoe** [comment]
@erezrokah thank you for the contribution 👍 it's exciting to see netlify using release-please 😄 

I'll get a release of the action out as soon as possible.

**Comment by erezrokah** [comment]
> @erezrokah thank you for the contribution 👍 it's exciting to see netlify using release-please 😄

We love it ❤️ https://github.com/search?q=org%3Anetlify+release-please&type=code

**Pull request merged**